sechub-analyzer-cli/src/test/java/com/daimler/sechub/analyzer/core/CommentCheckerCommentInLineTest.java

```diff
@@ -0,0 +1,389 @@
+package com.daimler.sechub.analyzer.core;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class CommentCheckerCommentInLineTest {
+    private static final String NOSECHUB = "NOSECHUB";
+    private static final String NOSECHUB_END = "END-NOSECHUB";
+    
+    private CommentChecker commentChecker;
+    
+    @Parameters
```

Comment:
Very nice!

I am always was to lazy to use parameters - but here it makes absolute sense. Nice work!